src/target/km1m7xx.c

```diff
@@ -0,0 +1,1965 @@
+/***************************************************************************
+ *   Copyright (C) 2005 by Dominic Rath                                    *
+ *   Dominic.Rath@gmx.de                                                   *
+ *                                                                         *
+ *   Copyright (C) 2006 by Magnus Lundin                                   *
+ *   lundin@mlu.mine.nu                                                    *
+ *                                                                         *
+ *   Copyright (C) 2008 by Spencer Oliver                                  *
+ *   spen@spen-soft.co.uk                                                  *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>. *
+ *                                                                         *
+ *                                                                         *
+ *   Cortex-M3(tm) TRM, ARM DDI 0337E (r1p1) and 0337G (r2p0)              *
+ *                                                                         *
+ *-------------------------------------------------------------------------*
+ *                                                                         *
+ *   This file is based on cortex_m.c and adds functionality for the       *
+ *   Nuvoton KM1M7 series.This file was created based on cortex_m.c.       *
+ *                                                                         *
+ *   Copyright (C) 2021 by Nuvoton Technology Corporation Japan            *
+ *   yamaguchi.yoshikazu@nuvoton.com                                       *
```

Comment:
Please add your full name prior to the mail address.

Reply:
Corrected to "Full name \<email address\>".